transport-native-epoll/src/main/java/io/netty/channel/epoll/Epoll.java

```diff
@@ -92,6 +93,26 @@ public static Throwable unavailabilityCause() {
         return UNAVAILABILITY_CAUSE;
     }
 
+    /**
+     * Returns {@code true} if the epoll native transport is both {@linkplain #isAvailable() available} and supports
+     * {@linkplain ChannelOption#TCP_FASTOPEN_CONNECT client-side TCP FastOpen}.
+     *
+     * @return {@code true} if it's possible to use client-side TCP FastOpen via epoll, otherwise {@code false}.
+     */
+    public static boolean isTcpFastOpenClientSideAvailable() {
```

Comment:
Consider referencing these new methods in javadoc for `Native.IS_SUPPORTING_TCP_FASTOPEN` to let users know that replacement is available.